committeeoversightapp/templates/committeeoversightapp/category_detail_page.html

```diff
@@ -0,0 +1,27 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <div class="col-12 col-lg-9 mx-auto">
+    <h1 class="mb-4">Category {{page.category.id}}: {{page.category.name}}</h1>
+  </div>
+
+  {% include 'partials/streamfield.html'%}
+
+  {% include 'partials/hearing_table.html' %}
+
+  <script>
+  $(document).ready(function () {
+      var id = {{page.category.id}}
+
+      $('#hearing-table').DataTable({
+        "dom": "Bfrtip",
+        "order": [[ 0, "desc" ]],
+        "columnDefs": [{ "orderable": false, "targets": [3, 4] }],
+        "processing": true,
+        "serverSide": true,
+        "ajax": "{% url 'order_list_json' 'category' %}/" + id
```

Comment:
Grab the category ID number and append `category/<id>` to the url serving the JSON for serverside DataTables loading